src/examples/gadgets.ts

```diff
@@ -24,8 +26,40 @@ console.timeEnd('compile');
 console.log('proving..');
 
 console.time('prove');
-let proof = await XOR.baseCase();
+let XORproof = await XOR.baseCase();
```

Comment:
tiny nit: this file starts getting pretty big, we could just combine all gadgets into one single `ZkProgram` named `BitwiseProver` (or something like that) and add individual methods to it